conf/examples/large-program.json

```diff
@@ -0,0 +1,21 @@
+{
+    "ana": {
+      "activated": [
+        "base", "mallocWrapper","escape"
```

Comment:
This is quite extreme, even deactivating `mutex`, which on any multithreaded program will probably produce a silly number of races. There's no criteria for being able to exclude any race.